recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ build:
 test:
     commands:
         - test -f ${PREFIX}/lib/libuuid.a  # [unix]
-        - conda inspect linkages -n _test libuuid  # [unix]
+        - conda inspect linkages libuuid  # [unix]
```

Comment:
Regarding the `not win` syntax, it is not about Windows but rather my "pet peeve" with the `unix` selector, both `Linux` and `OS X` are not `Unix` :smile: 

With that said. I don't really care to change the selector, it was just in the lines I copied. I only care about the `objects` if you think they are important.